src/main/java/org/testng/asserts/Assertion.java

```diff
@@ -743,4 +743,15 @@ public void doAssert() {
           }
         });
   }
+  
+  protected String getErrorDetails(Throwable error) {
```

Comment:
Should be static

Reply:
In fact, it can't be static if you want to use inheritance as expected.
You should add a comment and a test as sample => it will limit the risk of a modification

Reply:
Hmm that's correct.

Hmm, just a sample of what an expected error message should be as a test?

Reply:
The idea is just having a reference somewhere which explains how to use this feature.

It could be a test, a wiki page, or better a documentation update https://github.com/testng-team/testng-team.github.io

Reply:
The question to answer is how to 
 > be able to include a custom message in the SoftAssert.assertAll() call that would make tracing easier.